R/import_data.R

```diff
@@ -277,7 +276,7 @@ iNZread.dta <- function(obj, ...) {
                                        convert.dates   = TRUE,
                                        convert.factors = TRUE)
   
-  attr(temp.data.frame, "full.file") = TRUE
+  attr(temp.data.frame, "preview") = obj$preview
```

Comment:
No - the value of `attr(x, "preview")` doesn't actually matter - it should only be added if the file is a preview (the idea being that if anyone were to use this function then their resulting dataframe would have a random "preview = FALSE" attribute - they might not know what that means).